src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,97 @@
+package duke;
+
+import duke.command.ExitCommand;
+import javafx.application.Platform;
+import javafx.scene.image.Image;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents the main class for the Duke application
+ */
+
+public class Duke {
+    public static final String FILENAME = System.getProperty("user.dir") + "/data/duke.txt";
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+
+
+    /**
+     * Creates a new Duke object with the specified file path.
+     *
+     * @param filePath Path of the file.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            System.out.println(e.getMessage());
+            tasks = new TaskList();
+        }
+    }
+
+
+    /**
+     * Runs the duke application.
+     */
+    public void run() {
+        ui.showWelcome();
+        boolean isExit = false;
+        while (!isExit) {
+            try {
+                String fullCommand = ui.readCommand();
+                ui.showLine();
+                Command c = Parser.parse(fullCommand);
+                c.execute(tasks, ui, storage);
+                isExit = c.isExit();
+            } catch (DukeException e) {
+                ui.showError(e.getMessage());
+            } finally {
+                ui.showLine();
+            }
+        }
+    }
+
+
+    /**
+     * You should have your own function to generate a response to user input.
```

Comment:
Perhaps the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)